.swiftformat

```diff
@@ -0,0 +1,85 @@
+--acronyms ID,URL,UUID
```

Comment:
I'm on the fence about putting this in a dedicated module, so it is shared between all apps and can be updated in one place. What to you reckon ?

Reply:
I'm open to the idea. I tried to do it with an SPM that we could import as a dependency but the path of an SPM is hard to predict.
Maybe as git submodule ? But until now we only have had SPM so it could be tidious to always have to fetch both.